cartography/intel/aws/kms.py

```diff
@@ -187,6 +185,16 @@ def transform_kms_key_policies(
     policy_data = {}
 
     for key_id, policy, *_ in policy_alias_grants_data:
+        # Skip keys with null policy since we can't parse it
+        if policy is None:
+            # Add default empty policy data for keys with null policies
+            policy_data[key_id] = {
+                "kms_key": key_id,
+                "anonymous_access": False,
+                "anonymous_actions": [],
+            }
```

Comment:
Double check, but I think we should do this instead:

```suggestion
            policy_data[key_id] = {}
```

As currently written, this says that we know that this key has anonymous_access = False, but that's actually not true -- we just didn't have access to evaluate that. Instead, we should leave those values as None to indicate that we couldn't determine it.